jx/ui/html/HTMLRenderer.js

```diff
@@ -27,12 +27,11 @@ HTMLRenderer.prototype.flush = function() {
 	var dirtylist = this.dirtylist;
 
 	for (var uid in dirtylist) {
-		var attr = dirtylist[uid];
+		var delta = dirtylist[uid];
 
-		if (attr)
+		if (delta)
```

Comment:
`delta` should be the preferred convention over `attr` here since we are talking about a dirtylist. Correct me if I am wrong @yangbin